packages/proxy/package.json

```diff
@@ -40,6 +40,7 @@
     "@braintree/sanitize-url": "^7.0.1",
     "@types/yargs": "^17.0.32",
     "chalk": "^5.3.0",
+    "cheerio": "^1.0.0-rc.12",
```

Comment:
we should just ship it with the lib....

Reply:
malcolm-contrib yes

Reply:
once we make contrib real we can revert this